mariadb/templates/pvc.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if .Values.usePersistence }}
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: {{ template "fullname" . }}
+  annotations:
+    volume.alpha.kubernetes.io/storage-class: {{ .Values.dataVolume.storageClass | quote }}
```

Comment:
Does this work automatically in GKE? Or does a user need to create the storage class?